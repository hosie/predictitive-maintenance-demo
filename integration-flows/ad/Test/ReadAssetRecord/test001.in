{
  "d": {
         "name":"Spark Core",
         "v":"0.3",
         "vin":"1",
         "rssi":-68,
         "mf":76,
         "mb":73,
         "mt":"500",
         "t":"-",
         "tilt":"N"
       }
}

