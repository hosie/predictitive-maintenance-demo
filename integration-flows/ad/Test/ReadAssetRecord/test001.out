{"d":
 {
   "name":"Spark Core",
   "v":"0.3",
   "vin":"1",
   "rssi":-68,
   "mf":76,
   "mb":73,
   "mt":"500",
   "t":"-",
   "tilt":"N",
   "fittedParts":
   {
     "Data":
    [ 
      { 
        "DateFitted":"2014-11-18T00:00:00.000Z",
        "MilleageFitted":20000,
        "id":1,
        "vehicleId":1,
        "partId":1,
        "part":
        {  
          "SKU":"abc001",
          "description":"Fan belt",
          "StockLevel":0,
          "ServiceLimit":10000,
          "id":1
        }
      },
      {"DateFitted":"2013-10-09T23:00:00.000Z","MilleageFitted":10000,"id":2,"vehicleId":1,"partId":2,"part":{"SKU":"abc002","description":"Spark plug","StockLevel":5,"ServiceLimit":20000,"id":2}}]}}}